### PR TITLE
Showing BookDetails in seperate component

### PR DESCRIPTION
**Pull request by Ric0oo**
@szu-szy

### CODE REVIEW

File: src/components/BookDetails/BookDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+interface Params {
+  id: string;
+  [key: string]: string | undefined;
+}
+
+export const BookDetails: React.FC = () => {
+  const { id } = useParams<Params>();
+  const [book, setBook] = useState<BookType | null>(null);
+
+  useEffect(() => {
```
Comment (szu-szy) [inline src/components/BookDetails/BookDetails.tsx:16]: please move this fetch funcion above useEffect

**Review by szu-szy — COMMENTED** [review]
maly komentarz

File: src/components/BookDetails/BookDetails.tsx (hunk 1 above)
Comment (Ric0oo) [reply src/components/BookDetails/BookDetails.tsx]: Fixed

**Comment by szu-szy** [comment]
please resolve conflics

**Review by szu-szy — APPROVED** [review]

**Pull request merged**